src/components/ExternalLinkButton/__tests__/ExternalLinkButton.test.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { ExternalLinkButton } from '../index';
+import * as trackModule from '../../../utils/track';
+import userEvent from '@testing-library/user-event';
+
+describe('ExternalLinkButton', () => {
+  const component = (
+    <ExternalLinkButton
+      variation="primary"
+      size="small"
+      href="https://docs.aws.amazon.com/amplify/latest/userguide/getting-started.html"
+    >
+      Click Here!
+    </ExternalLinkButton>
+  );
+
+  it('should render the ExternalLinkButton component', async () => {
+    render(component);
+
+    const externalLinkButtonNode = await screen.findByText('Click Here!');
```

Comment:
Similar to the ExternalLink tests, can we use findByRole for this? Just to go a little further that it's actually outputting a link element. I think you can combine finding the text and role with 

```findByRole('link', { name: 'Click Here!'})```

since for a link it's name should be its content.

https://testing-library.com/docs/queries/byrole/